dev/devicelab/lib/framework/utils.dart

```diff
@@ -430,11 +430,12 @@ Future<String> eval(
   Map<String, String>? environment,
   bool canFail = false, // as in, whether failures are ok. False means that they are fatal.
   String? workingDirectory,
+  StringBuffer? stdout,
```

Comment:
Why do we need this change instead of using the return value?

Reply:
Because it throws an error on the `_execute` command and therefore never hits line 450 where it returns

Reply:
Ah, I see. Let's put the same comment here as from the line below, for consistency. (Unfortunate that this error output gets logged to stdout rather than stderr.)